# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-09-17 06:55+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: core/gql_queries.py:120 core/gql_queries.py:128 core/gql_queries.py:138
#: core/gql_queries.py:180 core/schema.py:505 core/schema.py:702
#: core/schema.py:716 core/schema.py:722 core/schema.py:730 core/schema.py:1085
#: core/utils.py:225 core/utils.py:230
msgid "unauthorized"
msgstr "User not authorized for this operation"

#: core/schema.py:1537
msgid "mutation.user_email_invalid"
msgstr ""

#: core/schema.py:1540
msgid "mutation.user_email_duplicated"
msgstr ""

#: core/schema.py:1542
msgid "mutation.user_no_email_provided"
msgstr ""

#: core/schema.py:1547
msgid "mutation.user_username_too_long"
msgstr ""

#: core/schema.py:1550
msgid "mutation.user_username_not_changeable"
msgstr ""

#: core/schema.py:1666
msgid "Failed to change user password"
msgstr ""

#: core/schema.py:1678
msgid "Username of the account to recover"
msgstr ""

#: core/schema.py:1693
msgid "Failed to reset password."
msgstr ""

#: core/schema.py:1704
msgid "Username of the user"
msgstr ""

#: core/schema.py:1707
msgid "Token used to validate the user"
msgstr ""

#: core/schema.py:1710
msgid "New password for the user"
msgstr ""

#: core/schema.py:1734
msgid "Failed to set password."
msgstr ""

#: core/services/userServices.py:38 core/services/userServices.py:144
#: core/services/userServices.py:186
msgid "core.user.edit_historical_data_error"
msgstr ""

#: core/services/userServices.py:242
msgid "core.wrong_old_password"
msgstr "The old password is incorrect"

#: core/services/userServices.py:259
msgid "Missing username or password"
msgstr ""

#: core/utils.py:175
msgid "core.string.over_max_length"
msgstr "Value %(value)s is over maximum length: %(max_length)s"

#: core/utils.py:194
msgid "core.insuree.unknown_gender"
msgstr "Unknown gender"

#: core/utils.py:196
msgid "core.insuree.unknown_dob"
msgstr "Unknown date of birth"

#: core/utils.py:361
msgid "core.utils.schema_validation.invalid_schema"
msgstr "Schema is not valid %(error)"

#: core/utils.py:365
msgid "core.utils.schema_validation.invalid_json"
msgstr "Schema json is not valid %(error)"

#: core/validation/objectExistsValidationMixin.py:6
#, python-format
msgid "%(model)s for id  %(id)s does not exists"
msgstr ""

#: core/validation/stringFieldValidationMixin.py:6
msgid "core.validation.stringFieldValidationMixin.empty_string"
msgstr "Field cannot be empty."

#: core/validation/stringFieldValidationMixin.py:7
msgid ""
"core.validation.stringFieldValidationMixin.string_starts_with_whitespace"
msgstr "Field cannot start with whitespace."

#: core/validation/stringFieldValidationMixin.py:8
msgid "core.validation.stringFieldValidationMixin.string_ends_with_whitespace"
msgstr "Field cannot end with whitespace."

#: core/validation/uniqueCodeValidationMixin.py:6
#, python-format
msgid "Object code %(code)s is not unique."
msgstr ""

#: core/views.py:38
msgid "Only user requesting export can fetch request"
msgstr ""
